web-client/src/views/AdvancedSearch/PractitionerSearchResults.tsx

```diff
@@ -1,73 +1,129 @@
+import { formatPositiveNumber } from '@shared/business/utilities/formatPositiveNumber';
+import { sequences, state } from '@web-client/presenter/app.cerebral';
+import React, { useRef } from 'react';
 
 export const PractitionerSearchResults = connect(
   {
-    MAX_SEARCH_RESULTS: state.constants.MAX_SEARCH_RESULTS,
-    advancedSearchHelper: state.advancedSearchHelper,
-    showMoreResultsSequence: sequences.showMoreResultsSequence,
+    PRACTITIONER_SEARCH_PAGE_SIZE:
+      state.constants.PRACTITIONER_SEARCH_PAGE_SIZE,
+    practitionerSearchHelper: state.practitionerSearchHelper,
+    submitPractitionerNameSearchSequence:
+      sequences.submitPractitionerNameSearchSequence,
   },
   function PractitionerSearchResults({
-    advancedSearchHelper,
-    MAX_SEARCH_RESULTS,
-    showMoreResultsSequence,
+    PRACTITIONER_SEARCH_PAGE_SIZE,
+    practitionerSearchHelper,
+    submitPractitionerNameSearchSequence,
   }) {
+    const paginatorTop = useRef(null);
+    const paginatorBottom = useRef(null);
     return (
       <>
-        {advancedSearchHelper.showSearchResults && (
+        {practitionerSearchHelper.showSearchResults && (
           <>
             <h1 className="margin-top-4">Search Results</h1>
-            {advancedSearchHelper.showManyResultsMessage && (
-              <WarningNotificationComponent
-                alertWarning={{
-                  message: 'Narrow your search by adding search terms.',
-                  title: `Displaying the first ${MAX_SEARCH_RESULTS} matches of your search.`,
-                }}
-                dismissible={false}
-                scrollToTop={false}
-              />
-            )}
-            <table className="usa-table search-results ustc-table responsive-table">
+            <div ref={paginatorTop}>
+              {practitionerSearchHelper.numberOfResults &&
+                practitionerSearchHelper.numberOfResults >
+                  PRACTITIONER_SEARCH_PAGE_SIZE && (
+                  <Paginator
+                    breakClassName="hide"
+                    forcePage={practitionerSearchHelper.activePage || 0}
+                    marginPagesDisplayed={0}
+                    pageCount={practitionerSearchHelper.pageCount || 0}
+                    pageRangeDisplayed={0}
+                    onPageChange={pageChange => {
+                      submitPractitionerNameSearchSequence({
+                        selectedPage: pageChange.selected,
+                      });
+                      focusPaginatorTop(paginatorTop);
+                    }}
+                  />
+                )}
+            </div>
+            <div className="text-right margin-bottom-2">
+              <span className="text-bold" id="custom-case-result-count">
+                Count: &nbsp;
+              </span>
+              <span data-testid="practitioner-search-result-count">
+                {formatPositiveNumber(
```

Comment:
I think any formatting should be done in the helper so we can test for it and to separate concerns a bit.

Reply:
Good call, will adjust.